packages/miniflare/src/workers/queues/schemas.ts

```diff
@@ -1,12 +1,15 @@
 import { Base64DataSchema, z } from "miniflare:zod";
 
+export const QueueMessageDelaySchema = z.number().int().min(0).max(43200).optional();
+
 export const QueueConsumerOptionsSchema = /* @__PURE__ */ z.object({
 	// https://developers.cloudflare.com/queues/platform/configuration/#consumer
 	// https://developers.cloudflare.com/queues/platform/limits/
 	maxBatchSize: z.number().min(0).max(100).optional(),
 	maxBatchTimeout: z.number().min(0).max(30).optional(), // seconds
 	maxRetires: z.number().min(0).max(100).optional(),
 	deadLetterQueue: z.ostring(),
+	retryDelay: QueueMessageDelaySchema,
```

Comment:
What does this represent? It looks like users can configure the delay when calling `.retry()`—why is it also specified in the consumer settings?

Reply:
Please refer to https://developers.cloudflare.com/queues/reference/batching-retries/#delay-on-retry
"Delays can also be configured in wrangler.toml...."